squarelet/organizations/forms.py

```diff
@@ -259,3 +271,91 @@ def clean(self):
         if good_organization and good_organization == bad_organization:
             raise forms.ValidationError("Cannot merge an organization into itself")
         return cleaned_data
+
+
+class ProfileChangeRequestForm(forms.ModelForm):
+    """Request changes to core organization profile data"""
+
+    url = forms.URLField(label=_("URL"), required=False)
+
+    class Meta:
+        model = ProfileChangeRequest
+        fields = ["name", "slug", "url", "city", "state", "country", "explanation"]
```

Comment:
Explanation should be a required field for non staff users when the form renders. This way, the browser can enforce the requirement before submission.